app/src/main/java/com/example/superimagesviewer/viewmodel/InstagramPostViewModel.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.superimagesviewer.viewmodel
+
+import android.app.Application
+import android.graphics.drawable.Drawable
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import com.example.superimagesviewer.repository.MosaicRepository
+
+class InstagramPostViewModel(application: Application) : AndroidViewModel(application) {
+
+    lateinit var photoUrl: String
```

Comment:
Not required, please pass this via `uploadImageToInstagram`

Reply:
done